### DISCUSSION TITLE
Stoped working with chrome after update

### DISCUSSION THREAD

**Discussion by MVL99** [aclap-dev/video-downloadhelper#255] [Help and Q&A]
### Your question

Hi,
Aftet updating to chrome to Version 122.0.6261.95 video downloader stopped working at all. The extensions shows,but when you click on it it does nothing.Any help would be appreciated

### Addon & CoApp version and details

Hi,
Aftet updating to chrome to Version 122.0.6261.95 video downloader stopped working at all. The extensions shows,but when you click on it it does nothing.Any help would be appreciated

**Comment by paulrouget** [comment]
Please [don't use caps](https://github.com/aclap-dev/video-downloadhelper/wiki/Code-Of-Conduct).

I can't help you without details. Please do [exactly what it says here](https://github.com/aclap-dev/video-downloadhelper/wiki/Where-can-I-find-my-versions%3F) and please share a failing URL.
